doc/dbext.txt

```diff
@@ -1,4 +1,4 @@
-*dbext.txt*	For Vim version 7.0.  Last change: 2016 Jan 03
+*dbext.txt*	For Vim version 7.0.  Last change: 2017 Nov 17
```

Comment:
Can you add a section at the bottom of this file around:
17.13 DBI Tutorial                            	*dbext-tutorial-dbi* 
Wishes shows this feature in action.
A command you execute and the output you get to see.

Reply:
Did some very "microscopic" documentation please have a look.